crates/bitwarden-core/src/auth/client_auth.rs

```diff
@@ -79,6 +81,11 @@ impl<'a> ClientAuth<'a> {
         make_register_tde_keys(self.client, email, org_public_key, remember_device)
     }
 
+    pub fn make_key_connector_keys(&self) -> Result<KeyConnectorResponse, CryptoError> {
```

Comment:
Why not return a `Result`? Equivalently, why does this method return a specific error type rather than the base error enum?

Reply:
We're transitioning away from a single massive enum as that meshes badly with the new crate structure. And it's generally a better experience knowing what errors you get vs needing to validate all errors.